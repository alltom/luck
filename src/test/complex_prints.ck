<<< "Countdown: 3 2 1 go!" >>>;
<<< "Countdown:", 3, 2, 1, "go!" >>>;
3 => int a;
<<< "Countdown:", a, a-1, a-2, "go!" >>>;
