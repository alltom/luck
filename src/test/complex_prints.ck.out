Countdown: 3 2 1 go!
Countdown: 3 2 1 go!
Countdown: 3 2 1 go!
